src/main.ts

```diff
@@ -160,6 +161,25 @@ ipcMain.handle('shell.openExternal', (event, data) => {
   return shell.openExternal(data)
 })
 
+ipcMain.handle(
+  'take.screenshot',
+  async (event, data: { width: number; height: number }) => {
+    const sources = await desktopCapturer.getSources({
+      types: ['window'],
+      thumbnailSize: { width: data.width, height: data.height },
+    })
+
+    for (const source of sources) {
+      if (source.name === 'Zoo Modeling App') {
+        // @ts-ignore image/png is real.
+        return sources[0].thumbnail.toDataURL('image/png') // The image to display the screenshot
```

Comment:
Oh this is a bug... I need to not do `sources[0]`. I was doing that for debugging, I need to use the `source` that I am iterating over.